ansible/roles/dnf_repos/defaults/main.yml

```diff
@@ -31,5 +31,15 @@ dnf_repos_repolist:
   name: extras
   base_url: "{{ dnf_repos_pulp_content_url }}/{{ appliances_pulp_repos.extras[ansible_distribution_version] | appliances_repo_to_subpath }}"
 
+dnf_repos_openhpc_repolist:
+- name: OpenHPC
+  file: OpenHPC
+  base_url: "{{ dnf_repos_pulp_content_url }}/{{ appliances_pulp_repos.openhpc_base[ansible_distribution_major_version] | appliances_repo_to_subpath }}"
+- name: OpenHPC-updates
+  file: OpenHPC
+  base_url: "{{ dnf_repos_pulp_content_url }}/{{ appliances_pulp_repos.openhpc_updates[ansible_distribution_major_version] | appliances_repo_to_subpath }}"
+
+dnf_repos_repolist: "{{ dnf_repos_default_repolist + dnf_repos_openhpc_repolist if openhpc_install_type == 'ohpc' else dnf_repos_default_repolist }}"
```

Comment:
I think `openhpc_install_type == 'ohpc'` needs a default filter as it will be undefined at present

Reply:
This sort of pattern is usually cleaner to do as:
```
full_list: "{{ list_a + (list_b if condition else [] ) }}"
```